components/intro-template.tsx

```diff
@@ -6,7 +6,14 @@ import introTemplateImg from '../images/introTemplateImg.png'
 export default function IntroTemplate() {
   const [studioURL, setStudioURL] = useState(null)
   const [createPostURL, setCreatePostURL] = useState(null)
+  const hasEnvVars =
+    process.env.NEXT_PUBLIC_VERCEL_GIT_PROVIDER &&
+    process.env.NEXT_PUBLIC_VERCEL_GIT_REPO_OWNER &&
+    process.env.NEXT_PUBLIC_VERCEL_GIT_REPO_SLUG
   const repoURL = `https://${process.env.NEXT_PUBLIC_VERCEL_GIT_PROVIDER}.com/${process.env.NEXT_PUBLIC_VERCEL_GIT_REPO_OWNER}/${process.env.NEXT_PUBLIC_VERCEL_GIT_REPO_SLUG}`
+  const removeBlockURL = hasEnvVars
```

Comment:
though I'm wondering if we should just put it always pointing towards our original repo to avoid this mess

Reply:
Yeah. Let's discuss this further tomorrow. Will merge this for now 👍